core/src/test/scala/org/apache/spark/SparkThrowableSuite.scala

```diff
@@ -568,7 +568,7 @@ class SparkThrowableSuite extends SparkFunSuite {
     assert(SparkThrowableHelper.getMessage(e2, STANDARD) ===
       """{
         |  "errorClass" : "UNSUPPORTED_SAVE_MODE.EXISTENT_PATH",
-        |  "messageTemplate" : "The save mode <saveMode> is not supported for: an existent path.",
+        |  "messageTemplate" : "The save mode <saveMode> is not supported. for an existent path.",
```

Comment:
This `supported. for an` looks weird. Could you improve the error message.

Reply:
Okay, let me think about it carefully.

Reply:
Done.